### PR TITLE
Log Create data

### PR DESCRIPTION
**Pull request by coziboy**
Add log for creating data

### CODE REVIEW

**Comment by haruncpi** [comment]
Create data generally not in log. Because 
-  It makes your `logs` table massy.
-  Created data already in your table.

**Comment by coziboy** [comment]
But I think some people need to record it into `logs` just to find out who was responsible for that data creation, that's why user can activate or disable it in config just in case they don't need to record every data creation

File: config/user-activity.php [hunk 1]
```diff
@@ -7,9 +7,10 @@
     'delete_limit'     => 7,
 
     'log_events' => [
+        'on_create'    => true,
```
Comment (haruncpi) [inline config/user-activity.php]: - Default value `false` make it better.
- Write the value with proper indentation.

**Review by haruncpi — CHANGES_REQUESTED** [review]

File: config/user-activity.php (hunk 1 above)
Comment (coziboy) [reply config/user-activity.php]: Fixed on https://github.com/haruncpi/laravel-user-activity/pull/11/commits/5309dc43dee07f17960d6c3866fcec9779bf4877

**Comment by haruncpi** [comment]
Please pull the latest master branch and just change the config and log trait without touching the `views/index.blade.php`  view file.

**Pull request closed without merging**